hnn_core/tests/test_dipole.py

```diff
@@ -98,3 +111,34 @@ def test_dipole_simulation():
     v_mask = vsoma > v_thresh
     assert np.all([spike_times[spike_gids == gid] > times[v_mask][0],
                    spike_times[spike_gids == gid] < times[v_mask][-1]])
+
+
+@requires_mpi4py
+def test_dipole_simulation_MPI():
+    """Test data produced from simulate_dipole() call across backends."""
+    hnn_core_root = op.dirname(hnn_core.__file__)
+    params_fname = op.join(hnn_core_root, 'param', 'default.json')
+    params = read_params(params_fname)
+    params.update({'N_pyr_x': 3,
+                   'N_pyr_y': 3,
+                   'tstop': 25,
+                   't_evprox_1': 5,
+                   't_evdist_1': 10,
+                   't_evprox_2': 20})
+    joblib_net = Network(params)
+    mpi_net = Network(params)
+    trial, n_trials, gid = 0, 2, 7
+    n_times = np.arange(0., params['tstop'] + params['dt'], params['dt']).size
+    with JoblibBackend(n_jobs=1):
+        simulate_dipole(joblib_net, n_trials=n_trials, record_vsoma=True,
+                        record_isoma=True)
```

Comment:
what is this guy doing in there ? copy-paste issue?

Reply:
I put that in there because a few lines down, we're comparing joblib and mpi. I assume this could be done better (taking joblib_net from the previous function?), but I didn't know how.